### ISSUE TITLE
Unable to reproduce car-196 benchmark results

### ISSUE THREAD

**Issue by sakuralggm** [PKU-ICST-MIPL/Finedefics_ICLR2025#12]
Hi, thanks for your impressive work!

I am currently trying to reproduce the results on the **car-196** benchmark. However, I obtained an accuracy of **23.65%**, which is significantly different from the **84.67%** reported in the paper.

I noticed that in the following file within the repository: `FOCI-Benchmark/results/mc/car-196--_data_hehulingxiao_code_Finedefics_ICLR2025_checkpoints_idefics2-8b-pretrain-qlora-merge-finetune-qlora-merge.json`

The recorded accuracy is 23.86%, which aligns very closely with my reproduction results.

Could you please advise on the correct evaluation method or script for car-196 to achieve the performance mentioned in the paper? I suspect I might be missing a specific configuration or looking at a different evaluation checkpoint.

Thanks for your help!

**Comment by hehulingxiao** [comment]
Thanks for following our work. 

Probably you used training images for testing, since the naming format of car-196 is the same for training and testing sets. For example, 00575.jpg occurs both in training and testing sets.

**Comment by sakuralggm** [comment]
> Thanks for following our work.
> 
> Probably you used training images for testing, since the naming format of car-196 is the same for training and testing sets. For example, 00575.jpg occurs both in training and testing sets.

Thank you for the quick clarification!

You are absolutely right. I checked my data and realized I was indeed using the training images due to the identical filenames. After switching to the correct testing set, I have successfully reproduced the results reported in the paper.

Thanks again for your help.
